modin/experimental/core/execution/native/implementations/hdk_on_native/expr.py

```diff
@@ -485,7 +485,7 @@ def invert(self) -> "OpExpr":
         OpExpr
             The resulting bitwise inverse expression.
         """
-        return OpExpr("~", [self], self._dtype)
+        return OpExpr("BIT_NOT", [self], self._dtype)
```

Comment:
Do we test this operation on HDK?

Reply:
Yes, there is a test - https://github.com/modin-project/modin/blob/master/modin/experimental/core/execution/native/implementations/hdk_on_native/test/test_dataframe.py#L2014 . Currently it's executed on arrow due to https://github.com/modin-project/modin/blob/master/modin/experimental/core/execution/native/implementations/hdk_on_native/expr.py#L1007, but this PR enables the HDK execution.